.github/workflows/create_version_bump_PR.yml

```
@@ -0,0 +1,133 @@
+            CURRENT_TRACER_VERSION="$(grep -o -e dd-trace-dotnet/releases/download/v[0-9]*\.[0-9]*\.[0-9]* dotnet/build-packages.sh | sed 's#dd-trace-dotnet/releases/download/v##')"
+            echo Current tracer version is: $CURRENT_TRACER_VERSION
+            
+            echo Bumping tracer version to $TRACER_VERSION
+            sed -i -e "s#dd-trace-dotnet/releases/download/v$CURRENT_TRACER_VERSION#dd-trace-dotnet/releases/download/v$TRACER_VERSION#" dotnet/build-packages.sh
+            echo Replaced release version in file.
+            PR_BODY="$PR_BODY Also updates the tracer url to use the last version of the .net Tracer: $TRACER_VERSION"
+          fi
+
+          CURRENT_AGENT_VERSION="$(grep -o -e agent-binaries-[0-9]*\.[0-9]*\.[0-9]*  dotnet/build-packages.sh | sed 's/agent-binaries-//')"
+          AGENT_VERSION=${{ steps.last_release.outputs.release }}
+          echo Current agent version is: $CURRENT_AGENT_VERSION
+          echo "Last Agent version: $AGENT_VERSION"
+
+          major=0
+          minor=0
+          build=0
+          agent_major=0
+          agent_minor=0
+          agent_build=0
+          # break down the version number into it's components
+          regex="([0-9]+).([0-9]+).([0-9]+)"
+          if [[ $CURRENT_AGENT_VERSION =~ $regex ]]; then
+            major="${BASH_REMATCH[1]}"
+            minor="${BASH_REMATCH[2]}"
+            build="${BASH_REMATCH[3]}"
+          fi
+          regex="([0-9]+).([0-9]+).([0-9]+)"
+          if [[ $AGENT_VERSION =~ $regex ]]; then
+            agent_major="${BASH_REMATCH[1]}"
+            agent_minor="${BASH_REMATCH[2]}"
+            agent_build="${BASH_REMATCH[3]}"
+          fi
+
+          if [[ $agent_major > $major ]]; then
+            updateTracer = 1
+          elif [[ $agent_minor > $minor ]]; then
+            updateTracer = 1
+          elif [[ $agent_build > $build ]]; then
+            updateTracer = 1
+          fi
+
+          if [[ $updateTracer == 1 ]]; then
+            echo Updating agent version
+            
+            sed -i -e "s/agent-binaries-$CURRENT_AGENT_VERSION/agent-binaries-$AGENT_VERSION/" dotnet/build-packages.sh
+            echo Replaced agent version in file.
+            PR_BODY="$PR_BODY Also updates the agent url to use the last version: $AGENT_VERSION"
+          fi
+
+          echo "::set-output name=version::$VERSION"
+          echo "::set-output name=tracer_version::$TRACER_VERSION"
+          echo "::set-output name=pr_body::$PR_BODY"
+          
+      - name: Create Pull Request
+        id: pr
+        uses: peter-evans/create-pull-request@v3.10.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: "version-bump-${{steps.versions.outputs.version}}"
+          commit-message: "[.Net Version Bump] Bump to v${{steps.versions.outputs.version}}"
+          delete-branch: true
+          title: "[.Net Version Bump] Bump to v${{steps.versions.outputs.version}}"
```

This is only a .NET Version bump if we bump the tracer version right? Otherwise it's just a "version bump"? We still want to auto run the tag and release so I would suggest something like

- `[Version Bump]` for a non-tracer bump version bump
- `[Version Bump] [.NET]` for a tracer bump version bump

Or just drop the .NET from the title here, given you describe it in the body now.
```suggestion
          commit-message: "[Version Bump] Bump to v${{steps.versions.outputs.version}}"
          delete-branch: true
          title: "[Version Bump] Bump to v${{steps.versions.outputs.version}}"
```

---

> This is only a .NET Version bump if we bump the tracer version right

Nope, that's not true. There are actually 2 extensions (java is version 0.1.0).
So we could be releasing the .net version only without changing the .Net tracer

---

Yes, exactly. But this code currently _assumes_ it's a .NET version bump 🤔

---

I'm confused 😄 In this PR you make the tracer version bump _optional_, so this PR already covers _both_ the .NET version bump _and_ the non-.NET version bump. But this stage assumes it's always .NET (which it may not be). 

Obviously not critical or anything, I'm just getting confused 😄

---

It all depends what you think hides behind this `.Net` label. The tracer? the extension?
This PR allows to update the .Net AAS extension and optionaly the .Net tracer. But it's .net in all cases, no?

---

Oh, I thought there was only one extension that worked for both .NET and Java. 

Standing down 🤐